### PR TITLE
chore: code reduction on doHash

### PR DESCRIPTION
**Pull request by colin-axner**

### CODE REVIEW

File: go/ops.go [hunk 1]
```diff
@@ -190,41 +190,17 @@ func doHash(hashOp HashOp, preimage []byte) ([]byte, error) {
...
-		return blakeHash.Sum(nil), nil
+		return hashBz(crypto.BLAKE2b_512, preimage)
 	case HashOp_BLAKE2S_256:
-		blakeHash := crypto.BLAKE2s_256.New()
-		_, err := blakeHash.Write(preimage)
-		if err != nil {
-			return nil, err
-		}
-		return blakeHash.Sum(nil), nil
-		// TODO: there doesn't seem to be an "official" implementation of BLAKE3 in Go,
```
Comment (colin-axner) [inline go/ops.go:226]: took the liberty of removing this comment. If desired, someone can open an issue

File: go/ops.go [hunk 2]
```diff
@@ -190,41 +190,17 @@ func doHash(hashOp HashOp, preimage []byte) ([]byte, error) {
 		return hashBz(crypto.RIPEMD160, preimage)
 	case HashOp_BITCOIN:
```
Comment (colin-axner) [inline go/ops.go:191]: wasn't mentioned in the audit, but I added code reduction here as well

**Review by damiannolan — APPROVED** [review]
LGTM! Thank you @colin-axner

**Comment by damiannolan** [comment]
Might need to push an empty commit to this to trigger CI required check!

**Pull request merged**